cypress/utils/utils.ts

```diff
@@ -1632,6 +1632,14 @@ export function doesExistText(str: string, toBePresent: boolean): void {
     }
 }
 
+export function doesExistButton(str: string, toBePresent: boolean): void {
+    if (toBePresent) {
+        cy.contains(button, str).should("exist");
+    } else {
+        cy.contains(button, str).should("not.exist");
+    }
```

Comment:
**nitpick:** 

`cy.contains(button, str).should(toBePresent ? 'exist' : 'not.exist');`

Reply:
yeah, nice. I will use it :+1:

Reply:
Implemented, thanks for idea